src/tools/OptionsSchema.json

```diff
@@ -407,6 +407,11 @@
         "targetArchitecture": {
           "type": "string",
           "description": "[Only supported in local macOS debugging] The architecture of the debuggee. This will automatically be detected unless this parameter is set. Allowed values are x86_64 or arm64."
+        },
+        "checkForDevCert": {
+          "type": "boolean",
+          "description": "When true, the presence of dev certificates will be checked by runing dotnet `dev-certs https --check`, and if that fails the user will be prompted to create dev certs by running `dotnet dev-certs https --trust`",
```

Comment:
>description [](http://example.com/codeflow?start=11&length=11)

How about:

When true, the debugger will check if the computer has a self-signed HTTPS certificate used to develop web servers running on https endpoints. This option does nothing on Linux, VS Code remote, and VS Code Web UI scenarios. If the HTTPS certificate is not found, the user will be prompted to install it.

Reply:
Probably also need a sentence about what the default is (maybe copy from debugger-launchjson.md)?